.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Description of the issue**
+
+**How to reproduce the issue**
+
+\`\`\`
+
+put code in code blocks
+
+\`\`\`
+
+\<details\>
+
+put long logs in details blocks
+
+\<\/details\>
+
+**Cirq version (optional)**
```

Comment:
Why make version optional? Not specifying the version potentially makes issues harder to reproduce. Also, for users reading or searching through our issues version is also a good guide to how far cirq may have changed since something was first reported (and is therefore a better indicator of staleness than dates on github posts).

Reply:
removed optional, mostly this was just to reduce friction.